package.yml

```diff
@@ -28,6 +28,7 @@ config:
                 - { namespace: mblock }
                 - { namespace: media_manager }
                 - { namespace: mform }
+                - { namespace: rexstan }
```

Comment:
Die rexstan-Werte aus `rex_config` (DB) sollten ins Repo synchronisiert werden.

Reply:
erstmal ja, bis es ein settings/import export gibt
https://github.com/redaxo/redaxo.org/pull/84#discussion_r1139301033

Reply:
Das hier sollten wir allerdings als Default nur aufnehmen, wenn rexstan relative Pfade in die Config schreiben würde.

Reply:
Für ydeploy wäre eigentlich angenehmer, wenn es einfach über den gewohnten Weg klappen würde hier. Und wir nicht einen separaten Umgang mit der config bräuchten (import/export).